docker/prod/seeder

```diff
@@ -3,12 +3,19 @@
 OUTPUT=$(echo "\dt" | psql `echo $DATABASE_URL | cut -d? -f1` 2>&1)
 
 if [[ "$OUTPUT" = "No relations found." ]]; then
-	echo "Initialising database and running seed..."
-	DISABLE_DATABASE_ENVIRONMENT_CHECK=1 bundle exec rake db:structure:load db:seed
```

Comment:
Changing this results in a double initialization of the env. However this only happens on the first startup, so I think we can ignore it.